SPHINXsys/src/shared/particles/solid_particles.cpp

```diff
@@ -240,13 +226,48 @@ namespace SPH {
 
 		size_t total_surface_particles = surface_particles.body_part_particles_.size();
 
+		//write Min Principal stress
+		// precision: 6 - higher precision because it depends on the E modulus
```

Comment:
please use /***/ format for informational comment; for the ease of recognition from commented code blocks

Reply:
okey